server/publications/collections/packages.js

```diff
@@ -69,10 +69,13 @@ function transform(doc, userId) {
 //
 //  Packages Publication
 //
-Meteor.publish("Packages", function (shopCursor) {
-  check(shopCursor, Match.Optional(Object));
+Meteor.publish("Packages", function (shopId) {
```

Comment:
From my checks as well, I don't see any references that can break (so far). All the "subscribes" seem to already be set to use shopId. So, I think this is good... any objections?  cc @aaronjudd